lib/dfe/analytics/tasks/import_entities.rake

```diff
@@ -2,6 +2,8 @@ namespace :dfe do
   namespace :analytics do
     desc 'Send Analytics events for the (allowlisted) state of all records in the database'
     task :import_all_entities, %i[batch_size] => :environment do |_, args|
+      return unless DfE::Analytics.enabled?
```

Comment:
Nitpick - Might be worth having a `puts`here writing to the console, with a message like:
```
DfE Analytics is not enabled - Ignoring import_all_entities
```

Reply:
Added this 👍